canopy/app/saplings/vanilla.js

```diff
@@ -14,8 +14,7 @@
  * limitations under the License.
  */
 
-window.invokeRegisteredApp = function(domNode, api) {
+window.invokeRegisteredApp = function(domNode) {
```

Comment:
This probably should be sqashed to remove the add commit.

Reply:
Good feedback, changes are in.